crates/ty_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -307,8 +307,9 @@ Using a `ParamSpec` in a `Callable` annotation:
 from typing_extensions import Callable
 
 def _[**P1](c: Callable[P1, int]):
-    reveal_type(P1.args)  # revealed: @Todo(ParamSpec)
-    reveal_type(P1.kwargs)  # revealed: @Todo(ParamSpec)
+    # TODO: Should reveal `ParamSpecArgs` and `ParamSpecKwargs`
+    reveal_type(P1.args)  # revealed: Unknown
+    reveal_type(P1.kwargs)  # revealed: Unknown
```

Comment:
_Ideally_ I feel like these would still be `@Todo` types, so that it's clear to users that we're aware of this limitation and that this isn't our intended final behaviour for this. But it's also fine to leave this as-is if it would be a lot of code to make these still be `@Todo` types.